index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Anja Schmidt's Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="mobile_container">
+      <!-- Nav Bar -->
+      <header>
+        <ul>
+          <li><a href="#" rel="noopener noreferrer">Welcome</a></li>
+          <li><img class="menu" src="images/menu.png" alt="Menu" /></li>
+        </ul>
+      </header>
```

Comment:
- [ ] Good job on implementing the `<header>` tag 👍 however the `<nav>` tag should still be used for this part of the code, it should look something like below example.
```
<header>
  <nav>
    ...
  </nav>
<header>
```